janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticSearchIndexTest.java

```diff
@@ -247,4 +247,14 @@ public void testInjestPipeline() throws Exception {
             assertEquals(1, tx.queryStream(new IndexQuery("ingestvertex", PredicateCondition.of(STRING, Cmp.EQUAL, "hello"))).count());
         }
     }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testMapKey2Field_IllegalCharacter() {
+        index.mapKey2Field("here is an illegal character: •", null);
```

Comment:
Others mentioned this previously, but replace the dot in all test usages with `REPLACEMENT_CHAR`

Reply:
@sharpau I would probably copy the character to a constant in this class, so that we can detect when the replacement char is mistakenly changed in the interface.

Reply:
@amcp Done